### PR TITLE
Update supl-ddf-file.md

### PR DESCRIPTION
**Pull request by Cheoljin**
Along with SUPL CSP MSDN document, I update SUPL DDF file. This is new for Vibranium release.

I have not updated "ms.date: 07/20/2018" with this PR. If necessary, please update it together.

### CODE REVIEW

**Comment by Cheoljin** [comment]
This is PR for the MSDN description change: 
image

**Comment by illfated** [comment]
> This is PR for the MSDN description change:
> image

It is easier to see or go to the PR by linking the PR page directly, like this: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4881
(`https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4881` or simply `#4881`)

File: windows/client-management/mdm/supl-ddf-file.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ The XML below is for Windows 10, version 1809.
                     <Replace />
                   </AccessType>
                   <DefaultValue>1</DefaultValue>
-                  <Description>Optional. Determines the version of the SUPL protocol to use. For SUPL 1.0, set this value to 1. For SUPL 2.0, set this value to 2. The default is 1.</Description>
+                  <Description>Optional. Determines the major version of the SUPL protocol to use. For SUPL 1.0.0, set this value to 1. For SUPL 2.0.0, set this value to 2. The default is 1. Refer to FullVersion to define minor verison and service indicator</Description>
```
Comment (illfated) [inline windows/client-management/mdm/supl-ddf-file.md]: Typo correction and grammar/readability improvements
```suggestion
                  <Description>Optional. Determines the major version of the SUPL protocol to use. For SUPL 1.0.0, set this value to 1. For SUPL 2.0.0, set this value to 2. The default is 1. Refer to FullVersion to define the minor version and the service indicator.</Description>
```

File: windows/client-management/mdm/supl-ddf-file.md [hunk 2]
```diff
@@ -174,6 +174,29 @@ The XML below is for Windows 10, version 1809.
...
               </Node>
+              <Node>
+                <NodeName>FullVersion</NodeName>
+                <DFProperties>
+                  <AccessType>
+                    <Get />
+                    <Replace />
+                  </AccessType>
+                  <DefaultValue>1.0.0</DefaultValue>
+                  <Description>Optional. Determines the full version (X.Y.Z where X, Y and Z are major version, minor version, service indicator, respectively) of the SUPL protocol to use. The default is 1.0.0. If FullVersion is defined, Version field is ignored</Description>
```
Comment (illfated) [inline windows/client-management/mdm/supl-ddf-file.md]: Grammatical/readability improvements:
```suggestion
                  <Description>Optional. Determines the full version (X.Y.Z where X, Y and Z are the major version, the minor version, and the service indicator, respectively) of the SUPL protocol to use. The default is 1.0.0. If FullVersion is defined, Version field is ignored.</Description>
```

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/client-management/mdm/supl-ddf-file.md [hunk 3]
```diff
@@ -174,6 +174,29 @@ The XML below is for Windows 10, version 1809.
...
               </Node>
+              <Node>
+                <NodeName>FullVersion</NodeName>
+                <DFProperties>
+                  <AccessType>
+                    <Get />
+                    <Replace />
+                  </AccessType>
+                  <DefaultValue>1.0.0</DefaultValue>
+                  <Description>Optional. Determines the full version (X.Y.Z where X, Y and Z are the major version, the minor version, and the service indicator, respectively) of the SUPL protocol to use. The default is 1.0.0. If FullVersion is defined, Version field is ignored.</Description>
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/supl-ddf-file.md]:
```suggestion
                  <Description>Optional. Determines the full version (X, Y, or Z where X, Y, and Z are the major version, the minor version, and the service indicator, respectively) of the SUPL protocol to use. The default is 1.0.0. If FullVersion is defined, Version field is ignored.</Description>
```

Comment (illfated) [reply windows/client-management/mdm/supl-ddf-file.md]: I suggest replacing "or" with `and` like this:
```cpp
                  <Description>Optional. Determines the full version (X, Y, and Z
```

Comment (Cheoljin) [reply windows/client-management/mdm/supl-ddf-file.md]: X.Y.Z is to show the format like 1.2.3 as example.

**Comment by Cheoljin** [comment]
Question: how to finish up this PR? Can I click "Close and comment" by myself or someone will do that per release? I'd like to understand the normal process in editing a document.

**Comment by illfated** [comment]
@Cheoljin : This PR (just as most other PRs) will be merged into the master branch when e0i (Onur) asks the document owner or manager to follow up, after we have finished discussing if this content needs more review or not. There is no need for further action from your side, besides answering questions (if any) before the final review and merge. We simply need to wait for this process to complete the normal way.

File: windows/client-management/mdm/supl-ddf-file.md (hunk 3 above)
Comment (illfated) [reply windows/client-management/mdm/supl-ddf-file.md]: True, also notice that `X.Y.Z` is kept in the sentence, just before the following description
```
where X, Y, and Z are the major version, the minor version, and the service indicator, respectively
```
so it still retains its original meaning. Thank you for adding this suggestion to your content.

**Comment by illfated** [comment]
@e0i : By the looks of this PR so far, it should be ready for final review of the document owner or manager.

**Comment by e0i** [comment]
@illfated Thank you for your assistance as always. Much appreciated.

**Comment by e0i** [comment]
@ManikaDhiman The PR that has been contributed by an internal MS member is available for review. Thanks.

**Comment by DaniHalfin** [comment]
The default branch for this repository has changed. Commenting to remember to merge this to the new default, if this PR gets merged.

**Comment by e0i** [comment]
@Cheoljin 

The default branch for this repository has changed from `master` to `public`. Please edit your pull request to change the merge target to `public` via the `Edit` button at the top right corner. Thanks.

**Comment by Cheoljin** [comment]
@e0i Changed the target from mater to public.

**Pull request closed without merging**